db/migrate/20150408150105_create_batches.rb

```diff
@@ -0,0 +1,9 @@
+class CreateBatches < ActiveRecord::Migration
+  def change
+    create_table :batches do |t|
+      t.datetime :processed_at
+
+      t.timestamps null: false
```

Comment:
This results in the same timestamps as [for AppointmentSummary](https://github.com/guidance-guarantee-programme/output/pull/99/files#diff-1acd2e7e27a227829d5d14a91c863bb6L20).